lgc/patch/Patch.cpp

```diff
@@ -378,6 +371,13 @@ void Patch::addOptimizationPasses(lgc::PassManager &passMgr, CodeGenOpt::Level o
   fpm.addPass(createFunctionToLoopPassAdaptor(std::move(lpm2), true));
   fpm.addPass(LoopUnrollPass(
       LoopUnrollOptions(optLevel).setPeeling(true).setRuntime(false).setUpperBound(false).setPartial(false)));
+#if LLVM_MAIN_REVISION && LLVM_MAIN_REVISION < 444780
```

Comment:
I suspect you could probably remove these guards now, although they don't do any harm.